Libraries/Image/resolveAssetSource.js

```diff
@@ -25,6 +25,7 @@ var AssetRegistry = require('AssetRegistry');
 var PixelRatio = require('PixelRatio');
 var Platform = require('Platform');
 var SourceCode = require('NativeModules').SourceCode;
+var assetPathUtils = require('../../local-cli/bundle/assetPathUtils');
```

Comment:
I like this (assuming it works). Before this change we had two places that compute image file name given set of its properties (platform, scale, etc.). I would recommend extracting this change in a separate PR for easier review